packages/commerce-sdk-react/src/components/StorefrontPreview/storefront-preview.test.tsx

```diff
@@ -117,9 +117,9 @@ describe('Storefront Preview Component', function () {
         const parameters = {basketId: '123'}
         const MockedComponent = ({enableStorefrontPreview}: {enableStorefrontPreview: boolean}) => {
             const apiClients = useCommerceApi()
-            getBasketSpy = jest.spyOn(apiClients.shopperBaskets, 'getBasket')
+            getBasketSpy = jest.spyOn(apiClients.shopperBaskets!, 'getBasket')
             useEffect(() => {
-                void apiClients.shopperBaskets.getBasket({parameters})
+                void apiClients.shopperBaskets!.getBasket({parameters})
```

Comment:
Since we're in a test and shopperBaskets in the test will always exist, asserting the reference "!" should be fine.

Reply:
Oh cool, I didn't know about `!` in typescript for asserting non-null. TIL